src/DGtal/topology/CubicalComplex.ih

```diff
@@ -603,12 +603,30 @@ bool
 DGtal::CubicalComplex<TKSpace, TCellContainer>::
 belongs( Dimension d, const Cell& aCell ) const
 {
-  ASSERT( d <= dimension );
```

Comment:
Why removing this check ? BTW, why not `d < dimenson` instead :octocat:

Reply:
Debugging mistake, during development of the `belongs` function accepting `PreCell`. Reverting. Thanks!

> BTW, why not d < dimenson instead
I would say because `d` can be equal to `dimension`

Reply:
Yes but `d` starts at 0 or am I mistaken?

Reply:
0 is a pointel, 1 is a linel, 2 a surfel and 3 a voxel. So it can be 3.

Reply:
Ho yes, sorry, I forgot that! :octocat: